templates/pesto/src/recipe-app.html

```diff
@@ -118,21 +126,28 @@
 
           <!-- nav menu -->
           <paper-listbox
-              selected="{{_pageData.page}}"
+              selected="[[_pageData.page]]"
```

Comment:
I understand why there needs to be a separate `_selectedPage` property, but why does this need to change to one-way bound `_pageData.page` with `<a>` links for navigation? This is an issue if a user clicks on the `<paper-item>` outside of the `<a>` (where the paper-item gets selected, but the link isn't clicked).